babel.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function(api) {
```

Comment:
`babel-loader` in the `karma.config.js` wasn't picking up the configuration in `.babelrc` but did work when I changed it to a `babel.config.js`. 🤷‍♂